tests/main/interfaces-cups-control/task.yaml

```diff
@@ -50,15 +50,15 @@ debug: |
 
 execute: |
     echo "Then the plug is disconnected by default"
-    snap interfaces -i cups-control | MATCH "^\- +test-snapd-cups-control-consumer:cups-control"
+    snap interfaces -i cups-control | MATCH '^\- +test-snapd-cups-control-consumer:cups-control'
 
     echo "When the plug is connected"
     snap connect test-snapd-cups-control-consumer:cups-control
 
     echo "Then the snap command is able to print files"
-    echo "Hello World" > $TEST_FILE
-    test-snapd-cups-control-consumer.lpr $TEST_FILE
-    while ! test -e $HOME/PDF/*.pdf; do sleep 1; done
+    echo "Hello World" > "$TEST_FILE"
+    test-snapd-cups-control-consumer.lpr "$TEST_FILE"
+    while ! test -e "$HOME"/PDF/*.pdf; do sleep 1; done
```

Comment:
I don't thin this `while` is right. For example:
```sh
$ cd `mktemp -d`
/tmp/tmp.YRaUqbAI7p$ test -e foo*.pdf; echo $?
1
/tmp/tmp.YRaUqbAI7p$ touch foo_1.pdf
/tmp/tmp.YRaUqbAI7p$ test -e foo*.pdf; echo $?
0
/tmp/tmp.YRaUqbAI7p$ touch foo_2.pdf
/tmp/tmp.YRaUqbAI7p$ test -e foo*.pdf; echo $?
bash: test: foo_1.pdf: binary operator expected
2
```
that last one is the thing I think you don't want (and the reason for using `find | wc` above)